recipes/avrocpp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.8.2" %}
+{% set hash = "c17d84613187dda3611c9c95d6f3388036a330ba" %}
+{% set name = "avrocpp" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: avro-cpp-{{ version }}.tar.gz
+    url: https://archive.apache.org/dist/avro/avro-{{ version }}/cpp/avro-cpp-{{ version }}.tar.gz
+    sha1: {{ hash }}
+
+build:
+    number: 0
+    skip: true  # [win]
+
+requirements:
+    build:
+        - toolchain
+        - cmake
+        - snappy
+        - xz 5.2.*
+        - pkg-config  # [osx]
+        - zlib 1.2.*
+        - boost 1.66.*
+        - msinttypes  # [win]
+    run:
+        - xz 5.2.*
+        - snappy
+        - zlib 1.2.*
+        - boost 1.66.*
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libavrocpp_s.a  # [unix]
+        - conda inspect objects {{ name }}      # [osx]
+        - conda inspect linkages {{ name }}     # [linux]
+
+about:
+  home: http://hadoop.apache.org/avro
+  license: Apache 2.0
```

Comment:
Should also add `license_family: Apache` and `license_file: LICENSE`.

Reply:
Done 🏁